src/components/Welcomes/1b1.tsx

```diff
@@ -105,29 +129,53 @@ export default function Welcome({ back, restore }: Props) {
                 textColor="primary"
                 aria-label="icon tabs example">
                 <Tab icon={<FolderOpen />} aria-label={geti18nString('welcome_1b_tabs_backup')} />
-                {/* TODO: add support for Firefox */}
                 <Tab
-                    disabled={!('BarcodeDetector' in window || isWKWebkit) || !!isFirefox}
+                    disabled={!('BarcodeDetector' in window || isWKWebkit)}
                     icon={<Camera />}
                     aria-label={geti18nString('welcome_1b_tabs_qr')}
                 />
                 <Tab icon={<Text />} aria-label={geti18nString('welcome_1b_tabs_text')} />
             </Tabs>
-            <main className={classes.main}>
-                {tab === 0 ? FileUI() : null}
-                {tab === 1 ? isWKWebkit ? <WKWebkitQR onScan={restore} onQuit={() => setTab(0)} /> : QR() : null}
-                {tab === 2 ? TextArea() : null}
-
-                {tab === 0 ? (
+            {json && (
+                <Dialog onClose={clearJson} aria-labelledby="simple-dialog-title" open={json !== null}>
+                    <DialogTitle id="simple-dialog-title">{geti18nString('welcome_1b_hint_identity')}</DialogTitle>
+                    <List>
+                        {json!.whoami.map(identity => (
+                            <ListItem button onClick={() => void 0} key={identity.userId}>
+                                <ListItemAvatar>
+                                    <Avatar className={classes.avatar}>{true ? <PersonIcon /> : <CheckIcon />}</Avatar>
+                                </ListItemAvatar>
+                                <ListItemText primary={identity.nickname || identity.userId} />
+                            </ListItem>
+                        ))}
+                    </List>
+                    <DialogTitle id="simple-dialog-title">{geti18nString('welcome_1b_hint_network')}</DialogTitle>
+                    <List>
+                        {json!.grantedHostPermissions.map(host => (
+                            <ListItem button onClick={() => void 0} key={host}>
```

Comment:
if it is not ready for click now, just remove `button` and `onClick` properties

Reply:
It's reserved for click-exclude logic. But I don't quite know how to exclude one from current configuration.